apps/schema/src/components/ProfileManager.vue

```diff
@@ -0,0 +1,343 @@
+          <div>{{ table.description }}</div>
+          <ul class="list-group">
+            <li class="list-group-item nav-header">
+              <ButtonAlt @click="selectAll(table.name)">select all</ButtonAlt
+              ><ButtonAlt @click="deselectAll(table.name)"
+                >deselect all</ButtonAlt
+              >
+            </li>
+            <template v-for="column in table.columns">
+              <li
+                v-if="isVisible(column)"
+                class="list-group-item list-group-item-action"
+                :class="{ active: column.selected }"
+                @click="
+                  column.selected
+                    ? (column.selected = null)
+                    : (column.selected = true)
+                "
+              >
+                <h5>
+                  {{ column.name }} {{}}
+                  <sup
+                    ><span
+                      class="badge mr-1"
+                      :style="{ 'background-color': color(profile) }"
+                      v-for="profile in column.profiles"
+                      >{{ profile }}</span
+                    ></sup
+                  >
+                  <div class="float-right">
+                    <small class="align-right"
+                      ><a
+                        v-for="semantics in column.semantics"
+                        href="semantics"
+                        target="_blank"
+                        >{{ semantics }}</a
+                      ></small
+                    >
+                  </div>
+                </h5>
+
+                {{ column.description }}
+                <div>
+                  <small
+                    ><i>{{
+                      Object.keys(column)
+                        .filter((key) =>
+                          ["key", "refTableName", "columnType"].includes(key)
+                        )
+                        .map((key) => key + "=" + column[key])
+                        .join(", ")
+                    }}</i></small
+                  >
+                </div>
+              </li>
+            </template>
+          </ul>
+        </div>
+      </div>
+      <div class="col-2">
+        <h2>My selection:</h2>
+        <a href="" @click.prevent="downloadCsv()">download</a>
+        <ul>
+          <li v-for="table in selectedTables">
+            {{ table.name }}
+            <ul v-if="table.columns?.some((c) => c.selected)">
+              <li v-for="column in table.columns?.filter((c) => c.selected)">
+                {{ column.name }}
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style scoped>
+.list-group li:hover {
+  cursor: pointer;
+}
+</style>
+
+<script setup>
+import { ref, computed } from "vue";
+import { InputSearch, MessageError, ButtonAlt } from "molgenis-components";
+import VueScrollTo from "vue-scrollto";
+
+const error = ref(null);
```

Comment:
to make typing easier (and logic statements as well), wouldn't empty string be a better default value?